### PR TITLE
Feature: login form

### PR DESCRIPTION
**Pull request by OoOHAMiDOoO**
Add new login form into project:
...

### CODE REVIEW

File: file-2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (usaootestoousa) [inline file-2.txt:1]: It's better to use capital H.

File: file-2.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 hello
+a new line
```
Comment (usaootestoousa) [inline file-2.txt:2]: It's better to use capital A.

**Review by usaootestoousa — CHANGES_REQUESTED** [review]
Good feature, but minor issue in capitalize char.

**Review by usaootestoousa — APPROVED** [review]
Every this good.

**Comment by OoOHAMiDOoO** [comment]
Every thing is good.

**Pull request merged**